ui/src/services/api.js

```diff
@@ -131,8 +131,8 @@ export async function deployNode(url, token, node) {
         fun: 'state.orchestrate',
         arg: ['metalk8s.orchestrate.deploy_node'],
         kwarg: {
-          saltenv: 'metalk8s-2.0',
-          pillar: { orchestrate: { node_name: node }}
+          saltenv: `metalk8s-${version}`,
```

Comment:
Hmh, not really, this should be the 'version of the cluster'. Also, I think we should default the version of a node to the 'version of the cluster' at that point in time.

However.

We don't have such things as 'version of the cluster' stored anywhere at this point in time :thinking: